docs/specifications/tests/Syntax-TP/syntax06.md

```diff
@@ -1,54 +1,114 @@
-## SYNTAX06: No illegal characters in the SOA RNAME field
+# SYNTAX06: No illegal characters in the SOA RNAME field
 
-### Test case identifier
-**SYNTAX06** There must be no illegal characters in the SOA RNAME field
+## Test case identifier
+**SYNTAX06**
 
-### Objective
+## Objective
 
 The SOA RNAME field is a mailbox address. The SOA RNAME field is defined
-in section 3.3 of [RFC 1034](https://tools.ietf.org/rfc/rfc1034.txt) and
-section 2.2 of [RFC 1912](https://tools.ietf.org/rfc/rfc1912.txt). The RNAME
-field should follow the rules of an e-mail address also defined in section
-3.4.1 of [RFC 2822](https://tools.ietf.org/html/rfc2822#section-3.4).
+in section 3.3.13 of [RFC 1035] and section 2.2 of [RFC 1912]. The RNAME
+field should follow the rules of an e-mail address also defined in 
+[RFC 5322, section 3.4.1].
 
-### Inputs
+## Inputs
 
-The domain name to be tested.
+* "Child Zone" - The domain name to be tested.
 
-### Ordered description of steps to be taken to execute the test case
+## Ordered description of steps to be taken to execute the test case
 
-1. Obtain a set of name server IP addresses using [Method4] and [Method5].
-2. Create a SOA query for the zone.
-3. Send the SOA query over UDP to each name server IP address until a
-   response is received or until the set is exhausted.
-4. Get the RNAME from the SOA record.
-5. Convert the first non-escaped "." to an "@" in the RNAME.
-6. De-escape the RNAME, converting a "\." to ".".
-7. Verify the result from step 3 and 4 according to the Addr-spec
-   specification in RFC 2822 section 3.4.1. If this verification fails,
-   this test case failes.
+1. Obtain the set of name server IP addresses using [Method4] and [Method5]
+   ("Name Server IP").
 
+2. Create a SOA query for the apex of the *Child Zone* with "rd" flag unset.
 
-### Outcome(s)
+3. For each name server in *Name Server IP* do:
+   1. Send the SOA query over UDP to the name server.
+   2. If the name server does not respond, emit *[NO_RESPONSE]* and go
+      to next server.
+   3. If the name server responds with DNS response but does not 
+      include a SOA record in the answer section, emit 
+      *[NO_RESPONSE_SOA_QUERY]* and go to next server.
```

Comment:
There's no explicit requirement for an AA flag here. Is that correct?

Also. There's a lot going on here. Do you think you could split this big step into smaller ones?